src/app/components/App/Form.tsx

```diff
@@ -58,17 +70,29 @@ const FormSchema = Yup.object().shape({
 });
 
 const Form: React.FC = () => {
+  const [, setFileSelected] = React.useState<File>();
+  const [imagePreview, setImagePreview] = React.useState<string>('');
+  console.log(imagePreview);
+
   return (
     <FormContainer>
       <div css={formStyle}>
         <Formik
           initialValues={{email: '', name: '', info: '', birthdate: new Date()}}
           validationSchema={FormSchema}
           onSubmit={(values, {setSubmitting}) => {
-            setTimeout(() => {
-              alert(JSON.stringify(values, null, 2));
-              setSubmitting(false);
-            }, 400);
+            axios.post('http://localhost:8081/attendees', {
+              id: 'uuid' + Math.floor(Math.random() * 1000),
```

Comment:
better is to use some library for uuid as naive implementation will create collisions for bigger datasets